src/alfasim_score/conftest.py

```diff
@@ -1,9 +1,36 @@
 import pytest
 from pathlib import Path
 
+from alfasim_score.converter.alfacase.base_operation import BaseOperationBuilder
+from alfasim_score.converter.alfacase.convert_alfacase import ScoreAlfacaseConverter
 from alfasim_score.converter.alfacase.score_input_reader import ScoreInputReader
 
 
 @pytest.fixture
-def score_input_example(shared_datadir: Path) -> ScoreInputReader:
-    return ScoreInputReader(shared_datadir / "score_input_example.json")
+def score_input_gas_lift(shared_datadir: Path) -> ScoreInputReader:
```

Comment:
I would rather put the fixtures in the `fixtures.py` file and add it here

Reply:
What do you think @lincrosenbach ?

Reply:
By putting on fixtures.py it can be imported by other projects. On conftest.py it can only be use inside this project.

Reply:
I have plans to do this in the [PWPA-1963](https://esss.atlassian.net/browse/PWPA-1963).